tools/ashlar/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "ashlar" %}
```

Comment:
is this file needed here? I guess this should go to conda-forge or bioconda? I can help you with that, just ping me on the PR.

Reply:
bioconda/bioconda-recipes#35846

Reply:
on it